lib/graphql_devise/model/with_email_updater.rb

```diff
@@ -0,0 +1,49 @@
+module GraphqlDevise
+  module Model
+    class WithEmailUpdater
+      def initialize(resource, attributes)
+        @attributes = attributes
+        @resource   = resource
+      end
+
+      def call
+        resource_attributes = @attributes.except(:schema_url, :confirmation_success_url)
```

Comment:
What about something like this?
```suggestion
        new_email              = @attributes.delete(:email)
        resource_attributes = @attributes.except(:schema_url, :confirmation_success_url)
        
         if email.present?
```
I that in that way we wont have to play with `unconfirmed_email`

Reply:
problem with that is that I will have to merge it later when I assign attributes. I still want to update all the params together so I can make use of the model's validations.